app/src/main/java/rectangledbmi/com/pittsburghrealtimetracker/ui/serverdown/ServerDownDialogFragment.kt

```diff
@@ -0,0 +1,44 @@
+package rectangledbmi.com.pittsburghrealtimetracker.ui.serverdown
+
+import android.app.Dialog
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.support.v7.app.AlertDialog
+import android.support.v7.app.AppCompatDialogFragment
+
+import rectangledbmi.com.pittsburghrealtimetracker.R
+
+/**
+ * Dialog to tell users that the app is down.
+ * Created by epicstar on 5/23/17.
+ * @since 78
+ */
+
+class ServerDownDialogFragment : AppCompatDialogFragment() {
+
+    override fun onCreateDialog(savedInstance: Bundle?): Dialog =
+            AlertDialog.Builder(context!!, R.style.Theme_AppCompat_Dialog_Alert).run {
```

Comment:
please use `requireContext()` instead of `context!!`

Reply:
done.